docs/modern/RefetchContainer.md

```diff
@@ -85,12 +125,12 @@ module.exports = createRefetchContainer(
           }
         }
       }
-    `
+  `,
   },
   graphql.experimental`
-    query FeedStoriesRefetchQuery($count: Int) {
-      feed {
-        ...FeedStories_feed @arguments(count: $count)
+    query FeedStoriesQuery($offset: Int, $limit: Int) {
+      viewer {
+        ...FeedStories_viewer @arguments(offset: $offset, limit: $limit)
```

Comment:
We discourage using offset/limit style APIs, so I'd like to avoid using them in our documentation if possible.